### DISCUSSION TITLE
possible cmds additions

### DISCUSSION THREAD

**Discussion by N3C2L** [madprops/grasshopper#67] [Ideas]
I would be glad if you could add these 2 cmds:

"Custom Filters" command to the palette:

I love that you can set your own filter commands under settings > filter > custom filters. However the added filters from there appear only when clicked on the top filter button + scrolling all the way down to "custom". I would really appreciate if you could add the "custom filters" cmd to the palette to spawn the custom filters with this command.

"Toggle Tags" command to the palette:

Could be very useful, giving space for a better tab list overview. Tags toggled on/off to enable or hide them in the tabs list. Useful especially when tags are set to below or above the tabs.

**Comment by madprops** [comment]
Added `Custom Filters` and `Toggle Taglist`.

**Comment by N3C2L** [comment]
Much obliged. This is very handsome.

**Comment by N3C2L** [comment]
I think this 2 cmds would be great: Close all tabs that shares the same exact URL or domain as the initial tab.

**Comment by N3C2L** [comment]
This ideas are no longer relevant, so I can close this thread.
